src/OrleansEventSourcing/LogStorage/LogViewAdaptor.cs

```diff
@@ -279,7 +279,7 @@ protected class UpdateNotificationMessage : INotificationMessage
             /// <inheritdoc/>
             public override string ToString()
             {
-                return string.Format("v{0} ({1} updates by {2}) etag={2}", Version, Updates.Count, Origin, ETag);
+                return string.Format("v{0} ({1} updates by {2}) etag={3}", Version, Updates.Count, Origin, ETag);
```

Comment:
I'm OK checking this in, nevertheless for string format calls (where you are eagerly formatting the string, which is not always the case when logging), it's better to replace them with interpolated strings: `$"v{Version} ({Updates.Count} updates by {Origin}) etag={ETag}"`